docs/targets/infrajs.md

```diff
@@ -0,0 +1,156 @@
+# InfraJS Target
+
+This event target does programatic transformations on received [CloudEvents][ce] using a Javascript function.
+
+## Prerequisites
+
+Basic knoledge of Javascript ES5 is required. Please note there are limitations to the interpreter:
+
+- ES6 is not supported.
+- Regular expressions are not fully compatible with ES5 specification.
+- No imports are allowed.
+- No libraries are pre-loaded.
+
+## Create the InfraJS Target
+
+Create an instance of the InfraJS Target at TriggerMesh as part of a Bridge.
+
+- `Name` is an internal identifier for the target.
+- `JS Script` is the function that contains the event manipulation.
+- `Timeout`  is the amount of milliseconds to wait before timing out the script.
+- `Type Loop Protection` when enabled will raise an error if the returned CloudEvent type is the same as the incoming one.
+
+### JS Script
+
+The JS Script field must include a function named `handle` with a single parameter for the incoming CloudEvent, and return the outgoing CloudEvent.
+
+- The incoming parameter is the [JSON][ce-json] representation of the incoming event.
+- The incoming parameter can be re-used as the outgoing parameter. That is useful if we want to modify a small set of the values in the incoming event.
+- The outgoing event must be a JSON structure that contains [context attributes][ce-context-attributes] at the root and [data payload][ce-data] at the `data` element. See [JSON format reference][ce-json].
```

Comment:
`ce-context-attributes` is  not defined and so is `ce-data` and `ce-json`

Reply:
thanks
https://github.com/triggermesh/docs/pull/80/commits/26edd9c10bc044187f7e8a0b45d234c7e0a88b59